examples/hx711_driver_calibration/main.c

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I don't think an example application is needed for this. The test application should be enough.